pkg/distributor/distributor.go

```diff
@@ -337,8 +363,6 @@ func (d *Distributor) Push(ctx context.Context, req *client.WriteRequest) (*clie
 				Samples: samples,
 			},
 		})
-
-		numSamples += len(ts.Samples)
```

Comment:
Not sure its safe to remove this; we previously didn't count invalid samples for instance.

Reply:
fixed, we count total samples and valid samples now